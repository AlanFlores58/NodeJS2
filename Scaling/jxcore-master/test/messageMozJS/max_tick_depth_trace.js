// Copyright & License details are available under JXCORE_LICENSE file


var common = require('../common');

process.maxTickDepth = 10;
process.traceDeprecation = true;
var i = 20;
process.nextTick(function f() {
  console.error('tick %d', i);
  if (i-- > 0)
    process.nextTick(f);
});