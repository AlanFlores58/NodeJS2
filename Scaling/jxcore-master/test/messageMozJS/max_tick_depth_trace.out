tick 20
tick 19
tick 18
tick 17
tick 16
tick 15
tick 14
tick 13
tick 12
tick 11
Trace: (node) warning: Recursive process.nextTick detected. This will break in the next version of node. Please use setImmediate for recursive deferral.
    at Console.prototype.trace@console.js:*:*
    at maxTickWarn@node.js:*:*
    at _nextTick@node.js:*:*
    at nextTick@node.js:*:*
    at onwrite@_stream_writable.js:*:*
    at WritableState/this.onwrite@_stream_writable.js:*:*
    at Socket.prototype._write@net.js:*:*
    at doWrite@_stream_writable.js:*:*
    at writeOrBuffer@_stream_writable.js:*:*
Trace: (node) warning: Recursive process.nextTick detected. This will break in the next version of node. Please use setImmediate for recursive deferral.
    at Console.prototype.trace@console.js:*:*
    at maxTickWarn@node.js:*:*
    at _nextTick@node.js:*:*
    at nextTick@node.js:*:*
    at onwrite@_stream_writable.js:*:*
    at WritableState/this.onwrite@_stream_writable.js:*:*
    at Socket.prototype._write@net.js:*:*
    at doWrite@_stream_writable.js:*:*
    at writeOrBuffer@_stream_writable.js:*:*
tick 10
tick 9
tick 8
tick 7
tick 6
tick 5
tick 4
tick 3
tick 2
tick 1
Trace: (node) warning: Recursive process.nextTick detected. This will break in the next version of node. Please use setImmediate for recursive deferral.
    at Console.prototype.trace@console.js:*:*
    at maxTickWarn@node.js:*:*
    at _nextTick@node.js:*:*
    at nextTick@node.js:*:*
    at onwrite@_stream_writable.js:*:*
    at WritableState/this.onwrite@_stream_writable.js:*:*
    at Socket.prototype._write@net.js:*:*
    at doWrite@_stream_writable.js:*:*
    at writeOrBuffer@_stream_writable.js:*:*
tick 0
